src/pointer-analysis/value_set.h

```diff
@@ -90,70 +90,15 @@ class value_sett
   /// offsets (`offsett` instances). This is the RHS set of a single row of
   /// the enclosing `value_sett`, such as `{ null, dynamic_object1 }`.
   /// The set is represented as a map from numbered `exprt`s to `offsett`
-  /// instead of a set of pairs to make lookup by `exprt` easier. All
-  /// methods matching the interface of `std::map` forward those methods
-  /// to the internal map.
+  /// instead of a set of pairs to make lookup by `exprt` easier.
   class object_map_dt
+    : public std::map<object_numberingt::number_type, offsett>
```

Comment:
This basically is a revert of 3b5263ff. We should have really good reasons if that's what should happen.

Reply:
Ok I see. I will drop that commit for now and try to find a better solution